scripts/calculateEIGasCosts.ts

```diff
@@ -0,0 +1,127 @@
+import { ethers } from "hardhat";
```

Comment:
This is a script we used in development to compare the efficiency of :
1. Trading all of the input token into WETH and then from WETH into each of the components (using `1 + numComponents` number of 0x api quotes)
2. Trading directly between input token and the components. Letting 0x figure out the route to go for each trade. (Using `numComponent` number of api quotes). This is what we ended up implementing.

Please let me know if I should remove / move this script

Reply:
What were the discoveries of this investigation?